gsl_core/tests/test_models.py

```diff
@@ -0,0 +1,105 @@
+from django.core.exceptions import ValidationError
+from django.test import TestCase
+
+from gsl_core.models import Arrondissement, Departement, Perimetre, Region
+
+
+class PerimetreTest(TestCase):
+    def setUp(self):
+        # Création des régions
+        self.region_idf = Region.objects.create(insee_code="11", name="Île-de-France")
```

Comment:
Tu dois pouvoir utiliser les factories pour ça : 

```suggestion
        from .factories import RegionFactory
        self.region_idf = RegionFactory(insee_code="11", name="Île-de-France")
```

J'ai bien une RegionFactory.